### PR TITLE
fix some null annotation in ControlCatalog

### PR DESCRIPTION
**Pull request by workgroupengineering**
## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021019-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

File: samples/ControlCatalog/Pages/ClipboardPage.xaml.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ private void InitializeComponent()
 
         private async void CopyText(object sender, RoutedEventArgs args)
         {
-            await Application.Current.Clipboard.SetTextAsync(ClipboardContent.Text);
+            await Application.Current?.Clipboard?.SetTextAsync(ClipboardContent?.Text);
```
Comment (maxkatz6) [inline samples/ControlCatalog/Pages/ClipboardPage.xaml.cs:28]: As for me it should create another warning. Since you can't await a null object.

Comment (workgroupengineering) [reply samples/ControlCatalog/Pages/ClipboardPage.xaml.cs:28]: restored, I will fixing in other PR

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021053-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021054-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Pull request merged**